src/Microsoft.Identity.Client/Microsoft.Identity.Client.csproj

```diff
@@ -6,8 +6,8 @@
     <PackageLicenseUrl>https://go.microsoft.com/fwlink/?linkid=844762</PackageLicenseUrl>
     <PackageProjectUrl>https://go.microsoft.com/fwlink/?linkid=844761</PackageProjectUrl>
     <MinClientVersion>2.12</MinClientVersion>
-    <Description>This package contains the binaries of the Microsoft Authentication Library (MSAL).</Description>
-    <Summary>MSAL provides a library with easy to use authentication functionality for your .NET client on various platforms including Windows desktop, Windows Store, Xamarin iOS and Xamarin Android by taking advantage of Azure Active Directory and MSA.</Summary>
+    <Description>This package contains the binaries of the Microsoft Authentication Library (MSAL). MSAL provides easy to use authentication functionality for your .NET based client on various platforms including Windows desktop, Windows Store, Xamarin iOS and Xamarin Android by taking advantage of Azure Active Directory and MSA.</Description>
+    <Summary>This package contains the binaries of the Microsoft Authentication Library (MSAL). MSAL provides easy to use authentication functionality for your .NET based client on various platforms including Windows desktop, Windows Store, Xamarin iOS and Xamarin Android by taking advantage of Azure Active Directory and MSA.</Summary>
```

Comment:
Where are the _Description_ and _Summary_ used? Seems odd that they are identical.
Otherwise, LGTM